polaris-react/src/components/EmptyState/EmptyState.tsx

```diff
@@ -106,7 +106,12 @@ export function EmptyState({
   ) : null;
 
   const childrenMarkup = children ? (
-    <Text as="span" color="subdued" alignment="center">
+    <Text
+      as="span"
+      color="subdued"
```

Comment:
I just noticed something, the text in the figma isn't subdued, should we add a ternary here as well?